examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>I think we should keep the description of MovieLens short. The focus is on the dataloader and Movielens is our "hello world" example.</p><p> </p><p>I would call the headline "Downloading and perparing the dataset".</p><p> </p><p>I would just say that we use Movielens as an example how to use Merlin dataloaders.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>